library/src/main/java/kwasm/format/text/GlobalType.kt

```diff
@@ -20,17 +20,26 @@ import kwasm.format.text.token.Keyword
 import kwasm.format.text.token.Paren
 import kwasm.format.text.token.Token
 
+/**
+ * Parses a GlobalType from a list of Tokens.
+ * From [the docs](https://webassembly.github.io/spec/core/text/types.html#global-types):
+ *
+ * ```
+ *   globaltype ::=  t:valtype                => const t
+ *                   ‘(’ ‘mut’  t:valtype ‘)’ => var t
+ * ```
+ */
 fun List<Token>.parseGlobalType(currentIndex: Int): ParseResult<GlobalType> {
     val maybeOpenParen = this[currentIndex]
     return if (maybeOpenParen is Paren.Open) {
         val keyword = this[currentIndex + 1]
         if (keyword !is Keyword || keyword.value != "mut") {
-            throw ParseException("Invalid GlobalType: Expecting mut token", keyword.context)
+            throw ParseException("Invalid GlobalType: Expecting \"mut\"", keyword.context)
         }
```

Comment:
In my control instructions branch,  I've got an extension function that will interpret a keyword token IFF its value matches an argument... No need to wait for it though, we can refactor the stuff in this PR later to use it.